spec/unit/lib/cloud_controller/diego/bbs_apps_client_spec.rb

```diff
@@ -2,72 +2,95 @@
 
 module VCAP::CloudController::Diego
   RSpec.describe BbsAppsClient do
-    subject(:client) { BbsAppsClient.new(bbs_client) }
+    subject(:client) {         BbsAppsClient.new(bbs_client, config) }
```

Comment:
have you run `bake`?  the big whitespace here seems to indicate that `rubocop` wasn't run?

Reply:
We ran `bundle exec rake rubocop:auto_correct` multiple times, but this issue was not caught. We will remove the whitespaces manually.